File: Administrativo/spring-erp-servicio/net/royal/spring/core/dominio/filtro/FiltroPaginacionAniversario.cs
using net.royal.spring.framework.core.dominio;
using System;
using System.Collections.Generic;
using System.Text;

namespace net.royal.spring.core.dominio.filtro
{
    public class FiltroPaginacionAniversario
    {
        public Int32? IdPersona { get; set; }
        public String IdCentroCostos { get; set; }
        public DateTime? FechaInicio { get; set; }
        public DateTime? FechaFin { get; set; }

        public Int32? Mes { get; set; }
        public Int32? Dia { get; set; }
        public Int32? Anios { get; set; }
        public String Signo { get; set; }

        public String compania { get; set; }

        public ParametroPaginacionGenerico paginacion { get; set; }

        public FiltroPaginacionAniversario()
        {
            paginacion = new ParametroPaginacionGenerico();
        }
    }
}

File: Administrativo/spring-erp-web/SpringNg/src/app/erp_module/shared/selectores/empleado/dominio/FiltroPaginacionAniversario.ts
import { ParametroPaginacionGenerico } from '../../../dominio/ParametroPaginacionGenerico';
export class FiltroPaginacionAniversario {
    constructor() {
        this.paginacion = new ParametroPaginacionGenerico();
    }
    paginacion: ParametroPaginacionGenerico;

    idPersona: number;
    idCentroCostos: string;
    fechaInicio: Date;
    fechaFin: Date;

    mes: number;
    dia: number;
    anios: number;
    signo: string;

    compania: string;

}